ExamplePipeline.yml

```diff
@@ -49,6 +49,11 @@ jobs:
       requirements: '-r tests/requirements.txt'
       report: 'htmlmypy'
 
+  PublishTestResults:
```

Comment:
Apart from adding it to the example pipeline, it needs to be added to the README.

Reply:
Done in https://github.com/pyTooling/Actions/pull/13/commits/774ea7970e32cd1983d1fb57a41e1f43cc769347.